### DISCUSSION TITLE
Helping Build/Comtribute to Falinks Teambuilder

### DISCUSSION THREAD

**Discussion by Aguilar-Carlos** [txfs19260817/falinks-teambuilder#29] [Q&A]
Hello, 

My name is Carlos a current master's student in Applied Statistics. I am very fascinated and interested to contributing to your Falinks Teambuilder project. I am very skilled in data science and analytical statistics, I know how to code in R, and know a little bit of python but am looking to enhance my skills and experience. If there is any way that I can join the team and help create the best analytical pokemon site would be amazing, and would greatly appreciate the opportunity to help.

Thank you for your time. 

- Carlos

**Comment by txfs19260817** [comment]
Hello Carlos, Thank you for your interest in contributing to this project! You may find that this is a web application mainly built with Next.js, and unfortunately it is poorly documented. So I was thinking to firstly invite you to have a talk with @VGCPastes , then you can discuss and come up with ideas. For example, based on those teams we can ship more interesting data visualization graphs to users. Meanwhile, you may also try to run this project in your local environment and contribute some docs to it, which can be greatly helpful as well. I am looking forward to your joining and hearing ideas from you!

https://discord.gg/KFVG66FB

**Reply by Aguilar-Carlos** [reply]
Is it possible to get re-invited into the discord channel? It seems that the invitation has expired, please and thank you.

**Reply by txfs19260817** [reply]
Sure! https://discord.gg/nj6D43Pn @Aguilar-Carlos

**Comment by Aguilar-Carlos** [comment]
Thank you for your time, and I will reach out to @VGCPastes. Sorry for the
late reply, and I look forward to work with y'all soon.

**Comment by Aguilar-Carlos** [comment]
Thank you.
